apps/rpc/runtime.mjs

```diff
@@ -0,0 +1,21 @@
+import path from "node:path"
+import { URL } from "node:url"
+
+export async function resolve(specifier, context, nextResolve) {
```

Comment:
Rewrites `../index` to `../index.ts` for node imports. This means things work well with both next's build system and node --strip-experimental-types